pkg/tarmak/config/config.go

```diff
@@ -281,6 +281,38 @@ func (c *Config) AppendEnvironment(env *tarmakv1alpha1.Environment) error {
 	return c.writeYAML(c.conf)
 }
 
+func (c *Config) RemoveEnvironment(environment string) error {
+	for key, env := range c.conf.Environments {
+		if env.Name == environment {
+			c.conf.Environments = append(c.conf.Environments[:key], c.conf.Environments[key+1:]...)
+		}
+	}
+
+	for i := 0; i < len(c.conf.Clusters); i++ {
+		if c.conf.Clusters[i].Environment == environment {
+			c.conf.Clusters = append(c.conf.Clusters[:i], c.conf.Clusters[i+1:]...)
+			i -= 1
+		}
+	}
+
+	// Set currentCluster to another cluster if it is part of the environment we want to delete
+	currentConfigEnvironment, err := c.CurrentEnvironmentName()
+	if err != nil {
+		return err
+	}
+	// Pick the first found environment/cluster to replace currentCluster
+	if currentConfigEnvironment == environment {
+		for _, cluster := range c.conf.Clusters {
```

Comment:
Why can't we just use `	c.SetCurrentCluster(c.tarmak.Clusters()[0].ClusterName())` ?

Reply:
Can we have a function that lists all clusters in the config?